spec/models/nomis_court_hearing_spec.rb

```diff
@@ -21,7 +21,7 @@
     let(:expected_attributes) do
       {
         id: nomis_court_hearing['id'],
-        start_time: Time.parse(nomis_court_hearing['dateTime']),
+        start_time: Time.zone.parse(nomis_court_hearing['dateTime']),
```

Comment:
As this is also coming from Nomis, does the same caveat apply as above with `activity_spec` ?

Reply:
good question, it looks like this only applies to activity, as in the court hearing we are explicitly passing in the zone:
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/models/nomis_court_hearing.rb#L11

Reply:
Cool, thanks for checking 👍